### PR TITLE
Use `script.LocalValue` for arguments and `this`

### PR DESCRIPTION
**Pull request by jrandolf**
According to the steps, `script.LocalValue` must include `script.RemoteReference` and `script.ChannelValue`. A priori, there is no purpose in `script.ArgumentValue`

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/webdriver-bidi/pull/477.html" title="Last updated on Jul 10, 2023, 5:14 PM UTC (829f709)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webdriver-bidi/477/cbc17e9...829f709.html" title="Last updated on Jul 10, 2023, 5:14 PM UTC (829f709)">Diff</a>

### CODE REVIEW

File: index.bs
Comment (thiagowfx) [inline index.bs:4530]: It makes me wonder whether `LocalValue` should be renamed to `ArgumentValue` now (as it is effectively used as an argument).

**Review by thiagowfx — APPROVED** [review]
Approving, but please wait for review from Mozilla just to double check it's correct.

File: index.bs
Comment (jrandolf) [reply index.bs:4530]: LocalValue derives its name from the spec instructions.

File: index.bs
Comment (jgraham) [inline index.bs:4585]: I don't think this should be `local-cddl`, right? These values are never parsed by the client.

File: index.bs
Comment (jgraham) [inline index.bs:6012]: Also this.

**Review by jgraham — CHANGES_REQUESTED** [review]
I think this is OK. The main change seems to be that we allow clients to send maps with channels or remote values as arguments.

The description of `script.LocalValue` needs to be updated to match the change.

I think adding all these types to `local-cddl` is wrong; I don't think clients ever have to parse these values.

File: index.bs
Comment (jrandolf) [reply index.bs:4450]: It can't be since `script.LocalValue` exists in `local-cddl`.

File: index.bs
Comment (jrandolf) [reply index.bs:5193]: It can't be since `script.LocalValue` exists in `local-cddl`.

File: index.bs
Comment (jrandolf) [reply index.bs:6012]: It can't be since `script.LocalValue` exists in `local-cddl`.

File: index.bs
Comment (jrandolf) [reply index.bs:4585]: They aren't parsed, but they are passed by the client.

**Comment by jrandolf** [comment]
> I think adding all these types to local-cddl is wrong; I don't think clients ever have to parse these values.

Sure, but CDDL complains about unknown types if you don't include it because LocalValue references it and LocalValue exists in `local-cddl`.

**Comment by jrandolf** [comment]
As discussed offline, the solution is to just make `script.LocalValue` only defined in `remote.cddl`.

**Review by jgraham — APPROVED** [review]

**Pull request merged**
